### PR TITLE
Issue solving: adapt examples, check read/writeOnly

### PR DESCRIPTION
**Pull request by adriankast**
# Changes
* Added two new examples (former green colored examples enhanced with default values to really pass all validations)
* Changed classification of former valid examples to warning
* Removed `private: true` tag from package infos to prepare publication
* Added additional check for read/writeOnly:
  * Warning if `readOnly` and `writeOnly` are true
  * Warning if `writeOnly` and `observable` are true
  * Warning if `writeOnly` or `readOnly` is true and `op` not set (--> default is `["readproperty", "writeproperty"]`)
  * Warning if `writeOnly` or `readOnly` is set and `readproperty` or `writeproperty` contained in `op` array or is the `op` string value

# Solves
* #130
* #117 
* #116

### CODE REVIEW

File: playground-core/index.js [hunk 1]
```diff
@@ -314,6 +317,96 @@ function tdValidator(tdString, logFunc, { checkDefaults=true, checkJsonLd=true }
...
+                    const curPropertyName = tdProperties[i]
+                    const curProperty = td.properties[curPropertyName]
+
+                    // if readOnly is set
+                    if (curProperty.hasOwnProperty("readOnly") && curProperty.readOnly === true) {
+                        // check if both readOnly and writeOnly are true
+                        if (curProperty.hasOwnProperty("writeOnly") && curProperty.writeOnly === true) {
+                            details.readWriteOnly = "warning"
+                            logFunc('! Warning: In property ' + curPropertyName +
+                                ', both readOnly and writeOnly are set true!')
```
Comment (egekorkan) [inline playground-core/index.js:340]: set _to_ true :)

File: playground-core/index.js [hunk 2]
```diff
@@ -314,6 +317,96 @@ function tdValidator(tdString, logFunc, { checkDefaults=true, checkJsonLd=true }
...
+                                    }
+                                }
+                            }
+                        }
+
+                        // check if observable is also set
+                        if (curProperty.hasOwnProperty("observable") && curProperty.observable === true) {
+                            details.readWriteOnly = "warning"
+                            logFunc('! Warning: In property ' + curPropertyName +
+                                ', both writeOnly and observable are set true!')
```
Comment (egekorkan) [inline playground-core/index.js:404]: set _to_ true

**Review by egekorkan — CHANGES_REQUESTED** [review]
only super minor comments

File: playground-core/index.js (hunk 1 above)
Comment (adriankast) [reply playground-core/index.js]: Done https://github.com/thingweb/thingweb-playground/pull/131/commits/34e53654b6ecab630f69ce8ce760202a1da54179 :)

File: playground-core/index.js (hunk 2 above)
Comment (adriankast) [reply playground-core/index.js]: Done https://github.com/thingweb/thingweb-playground/pull/131/commits/34e53654b6ecab630f69ce8ce760202a1da54179 :)

**Pull request merged**